test/unit/specs/vislib/visualizations/tile_map.js

```diff
@@ -179,8 +180,10 @@ define(function (require) {
             var count = Math.random() * 50;
             var max = 50;
             var precision = 1;
-            var feature = chart.chartData.geoJson.features[0];
-            expect(_.isNumber(chart.radiusScale(count, max, feature))).to.be(true);
+            var mapData = chart.chartData.geoJson;
+            var i = _.random(0, mapData.features.length - 1);
+            var feature = mapData.features[i];
+            expect(_.isFinite(chart.radiusScale(count, max, feature))).to.be(true);
```

Comment:
Check the value of the returned function instead of whether or not it returns a finite number